localizer:

  # gates = [ x0, y0, z0, vx0, vy0, vz0, yaw0,
  #           x1, y1, z1, vx1, vy1, vz1, yaw1,
  #           ... ]   { yaw in radians, x y z in meters, vx vy vz in m/s (all in inertial coordinate frame)}
  #

  # Two Gates
  # gates: [ 7.5, -0.75, 2.25, 0.0, 0.0, 0.0, 0.0, 
  #         -7.5,  0.75, 2.25, 0.0, 0.0, 0.0, 3.1412 ]


  # Four Gates
  # gates: [ 5.0, 0, 2.25, 0.0, 0.0, 0.0, -1.57, 
  #          0.0, 5.0, 2.25, 0.0, 0.0, 0.0, -3.1412, 
  #          -5.0, 0, 2.25, 0.0, 0.0, 0.0, 1.57, 
  #          0.0, -5.0, 2.25, 0.0, 0.0, 0.0, 0.0 ]
  

  # Four GAtes Staggered
  gates: [ 9.0, 0.0, 2.25, 0.0, 0.0, 0.0, 0.0, 
          10.0, -9.0, 2.25, 0.0, 0.0, 0.0, -1.57, 
          1.0, -10.0, 2.25, 0.0, 0.0, 0.0, -3.14, 
          0.0, -2.0, 2.25, 0.0, 0.0, 0.0, 1.57 ]

  odomSubTopic: "/ground_truth/state"
  odomPubTopic: "/localizer/estimated_state"
  gatePoseSubTopic: "/gate_detector/raw_gate_pose"

  predict_rate: 60.0
  Q: [ 0.5, 0.5, 0.5 ]
  R: [ 0.1, 0.1, 0.1 ]